jenkins-integration/jenkins-jobs/common/scripts/jenkins/pipeline.groovy

```diff
@@ -403,6 +417,7 @@ def multipass_pipeline(jobs) {
                     SCRIPT_DIR,
                     server_era)
             step030_customize_settings(SCRIPT_DIR, job['puppet_settings'])
+            step035_customize_hocon_settings(SCRIPT_DIR, job['hocon_settings'])
```

Comment:
I think the agent is disabled on the master as part of the PE install.  During the "file_sync" setup step, though, we run the agent in order to have the file sync configuration applied.  If we were to move the `customize_hocon_settings` step to just before or after `step080_customize_settings`, I think we'd get the potential benefit of being able to modify a HOCON config setting that wouldn't be overridden by an agent run.  Although, admittedly, that's a bit precarious since someone could always add a step later on which runs the agent again and undoes the HOCON configuration.  I'd have a slight preference for moving this step to around where 080 is - and maybe document somewhere that future steps which do agent runs should not happen after the settings* steps.  But, again, I'd be good with punting on all of this if neither @jpinsonault nor @cprice404 thinks it's worth doing anything more on this at this point.